litestar/datastructures/headers.py

```diff
@@ -380,8 +382,8 @@ def from_header(cls, header_value: str) -> "ETag":
         except ValidationError as exc:
             raise ImproperlyConfiguredException from exc
 
-    @validator("value", always=True)
-    def validate_value(cls, value: Any, values: Dict[str, Any]) -> Any:
+    @field_validator("value")
+    def validate_value(cls, value: Any, values: Dict[str, Any], info: FieldValidationInfo) -> Any:
```

Comment:
```suggestion
    def validate_value(cls, value: Any, values: Dict[str, Any]) -> Any:
```
I dont think this is needed